api/stream/design.go

```diff
@@ -516,6 +516,22 @@ var _ = Service("stream", func() {
 		})
 	})
 
+	Method("ListSinkVersions", func() {
```

Comment:
(unrelated to this line) For later endpoints can I ask you to split the changes so that the generated code is in a separate commit from your own changes? It would make review a bit easier.